pkg/network/ovn_kubernetes.go

```diff
@@ -477,13 +491,24 @@ func bootstrapOVN(conf *operv1.Network, kubeClient client.Client) (*bootstrap.Bo
 		}
 	}
 
+	prePullerDS := &appsv1.DaemonSet{}
```

Comment:
We should probably add a wait loop to make sure the daemonset gets created before we remove it from the loop? Like stated in the previous comment the startup situation is a bit confusing.

Reply:
We don't need a wait loop if our logic is level-triggered. In other words, if we decide we want the pre-pull, it doesn't matter how many times CNO triggers a re-rendering pass, the output *should* be stable.

Reply:
hmm yea, I am unraveling the magic of level-triggering..